src/app/app.component.ts

```diff
@@ -64,6 +64,7 @@ export class AppComponent {
       */
 
     this.permissionsService.loadRolesAndPermissions({
```

Comment:
This role hierarchy could be loaded from the backend. Is it a temp list ?

Reply:
Yes, in a second step, we need to load that from the backend.